### PR TITLE
Handle missing opening times for a pharmacy

### PR DESCRIPTION
**Pull request by neilbmclaughlin**
This work is complete but there is an issue with test execution times (#39)

### CODE REVIEW

**Comment by neilbmclaughlin** [comment]
@st3v3nhunt In view of the outcome of performance meeting can we talk to decide what to do about this PR?

**Comment by st3v3nhunt** [comment]
@neilbmclaughlin The build is still failing. Changes LGTM

File: test/integration/app.js [hunk 1]
```diff
@@ -146,7 +146,8 @@ describe('The results routes', () => {
     const openResultsRoute = '/results-open';
 
     describe('happy paths', () => {
-      it('should return the top 2 open results when the postcode is valid', (done) => {
+      it('should return the top 2 open results when the postcode is valid', function (done) {
+        this.timeout(9000); // TODO: refactor to make this unecessary
```
Comment (neilbmclaughlin) [inline test/integration/app.js]: My guess is it won't be, I think there is something else going on.

**Comment by neilbmclaughlin** [comment]
@st3v3nhunt - build issues finally resolved (see comments for last commit for details) and code ready for merge.

**Comment by st3v3nhunt** [comment]
The commit history has a number of 'merge into master' commits. Can these be squashed to clean up the history please?

**Comment by neilbmclaughlin** [comment]
Commits squashed, tried to fix the emoji mistype (:wine:) with no luck so decided to leave it.

**Pull request merged**